packages/@lwc/engine-server/src/__tests__/fixtures/comments-foreach/expected.html

```diff
@@ -1,6 +1,7 @@
 <x-comments-foreach>
   <template shadowroot="open">
     <ul>
+      ‍
```

Comment:
these are because on ssr we transform empty texts into '\u200D'. the transformation is needed for empty expressions in the template, and now also makes things a bit easier during hydration.